packages/react-admin-form/src/Field.tsx

```diff
@@ -38,7 +38,7 @@ export class Field<FieldValue = any, T extends HTMLElement = HTMLElement> extend
             return (
                 <UsedFieldContainer label={label} required={required}>
                     {React.createElement(component, { ...rest, input, meta })}
-                    {meta.error && meta.touched && <Typography color="error">{meta.error}</Typography>}
+                    {(meta.error || meta.submitError) && meta.touched && <Typography color="error">{meta.error || meta.submitError}</Typography>}
```

Comment:
was, wenn error und submitError gesetzt sind? gibst du da bewusst nur error aus? oder kann das in der praxis eh nicht sein?

Reply:
es können beide gesetzt sein aber dann ist immer der error aktueller und soll angezeigt werden (Frontend validierung).